src/send.js

```diff
@@ -32,8 +35,14 @@ let sendDataPoints = async (configurer, options, payload) => {
       },
       encoding: null,
       body:     serialize(payload),
-      timeout:  30 * 1000,
-      retries:  5,
+      timeout:  90 * 1000,
+      retries:  (iter, error) => {
```

Comment:
on a worker where DHCP haven't kicked in yet wouldn't `ENETUNREACH` be transient?

Of course how would it then report to sentry :)